libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -9,10 +11,53 @@ import { getReadingList, removeFromReadingList } from '@tmo/books/data-access';
 })
 export class ReadingListComponent {
   readingList$ = this.store.select(getReadingList);
+  book:Book;
```

Comment:
Shouldn't need these transient class properties. Can you remove `book`, `readingItem`, and `readingItems` from the class?